src/test/java/com/marklogic/appdeployer/command/hosts/AssignHostsToGroupsTest.java

```diff
@@ -0,0 +1,38 @@
+package com.marklogic.appdeployer.command.hosts;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.marklogic.appdeployer.AbstractAppDeployerTest;
+import com.marklogic.appdeployer.ConfigDir;
+import com.marklogic.appdeployer.command.groups.DeployGroupsCommand;
+import com.marklogic.mgmt.resource.hosts.HostManager;
+
+public class AssignHostsToGroupsTest extends AbstractAppDeployerTest {
```

Comment:
Should it be possible for this to work on a single host? I'm leaning towards this just being a "Debug" test that is run manually against a 3-node cluster.